pstress/pstress-run.sh

```diff
@@ -103,6 +103,66 @@ kill_server(){
   { kill -$SIG ${MPID} && wait ${MPID}; } 2>/dev/null
 }
 
+capture_tables_and_rows(){
```

Comment:
Given server is under ddl/dml load. So some ddl/dml might complete before the crash and after this check. How we will identify that if the mismatch is due to ddl happened due to that?

Are we going to take any action in the case of mismatch, apart from printing in log file. 

Also add option to enable/disable this check.

Reply:
Hi @rahulmalik87 - I am aware of this issue. I have already posted a question on the channel for feedback/suggestions.
The driver script has no control over workload generated by pstress. So, yes, you are right we will have mismatches due to DDLs/DMLs after stats are captured and before server is killed.

I have created this PR for suggestions how we can handle this is in code.

One idea I have is, what if we add this capturing logic in pstress workload. Before exiting it can capture such data, but how are we going to store to to validate it after crash recovery will be again a question